db/V2__ref_tables.sql

```diff
@@ -145,6 +145,7 @@ CREATE TABLE pawtal_events (
   stk TEXT DEFAULT NULL,
   x_pathname TEXT DEFAULT NULL,
   x_vet_account_id TEXT DEFAULT NULL,
+  x_query_string TEXT NOT NULL DEFAULT '',
```

Comment:
I think the default should be NULL to be consistent with all the other x_ columns. The idea is that the column applies only to some event types. E.g. if we had an event_type ui.user.account.edited, then setting any value to query string does not make any sense.

Reply:
Updated in cb8079b4